spdockit-insights/insights-performance.md

```diff
@@ -47,7 +53,12 @@ The server overview dashboard shows all metrics that are collected for each serv
 
 - __SQL__ category – this category is assigned to all SQL servers in SharePoint farms. Each server in Always-On cluster is included as well. These metrics show the SQL server performance.
 
-If the metric is in warning or critical state the notification above the metric name will indicate the problem. It will tell how the metric crossed the threshold value. The status of each metric is calculated based on average in the last fifteen minutes. The status of metrics for which the warning or critical thresholds are not defined will not be calculated.
+If the metric is in warning or critical state the notification above the metric name will indicate the problem. It will tell how the metric crossed the threshold value. The status of each metric is calculated based on average in the last fifteen minutes. The status of metrics for which the warning or critical thresholds are not defined will not be calculated.  
+
+Hover the mouse over any chart and in __top right corner__ you will notice the maximize button. Click it to expand the chart over the entire screen. You can toggle __Show thresholds__ button to show or hide the treshold lines (both critical and warning).  
+When the treshold lines are active the chart will scale to accommodate both the treshold lines and the vaules. When the treshold lines are turned off, the scaling will be done accordingly - only to the chart values will be shown.   
```

Comment:
_accommodate_ seems needlessly complicated of a word, _fit_ might be better - a very soft suggestion, either is fine probably.

__ the scaling will be done accordingly - only to the chart values will be shown. __ second half seems out of place? __... the chart will be rescaled to optimally include only the collected chart values.__